priv/repo/migrations/20171020100846_add_checkin_date.exs

```diff
@@ -0,0 +1,17 @@
+defmodule Library.Repo.Migrations.AddCheckinDate do
+  use Ecto.Migration
+
+  def up do
+    alter table(:book_loans) do
+      add :checked_in, :naive_datetime
+      remove :queue
+    end
+  end
+
+  def down do
+    alter table(:book_loans) do
+      remove :checked_in, :naive_datetime
+      add :queue
```

Comment:
This will need to be switched around sorry. So for `remove`, you don't say the type, just `remove :checked_in`. And for the `queue`, you will need to include the type that it will be when it's re-added, i.e. back to what it was before.

Reply:
Oops my bad :+1: